plugins/rtmp-services/data/services.json

```diff
@@ -1684,11 +1684,17 @@
             ]
         },
         {
-            "name": "niconico, premium member (ニコニコ生放送 プレミアム会員)",
+            "name": "niconico (ニコニコ生放送)",
+            "more_info_link": "https://live.nicovideo.jp",
```

Comment:
The `more_info_link` is usually meant for documentation on "stream setup/configuration" and not the home page of the service. Do you have anything applicable?

Reply:
The page I had used for `stream_key_link` was actually a documentation page, so I’ve moved it to `more_info_link` instead.